nsv13/code/modules/power/stormdrive.dm

```diff
@@ -88,6 +88,13 @@ Control Rods
 
 #define MAX_CONTROL_RODS 5
 
+//Plutonium sludge
+
+#define PLUTONIUM_SLUDGE_RANGE 5
```

Comment:
Habit from working on another codebase since the range on the effect spawner were 'magic random numbers' and should be #define'd. Could remove the #define if demanded.